package.json

```diff
@@ -1,28 +1,18 @@
 {
-  "name": "stoplight-elements-apis",
-  "description": "Basic React app with embedded Stoplight Elements: StoplightAPI and Stoplight Project",
-  "author": {
-    "name": "Michael Miaskowski",
-    "email": "mmiask92@gmail.com"
-  },
-  "version": "0.2.0",
+  "name": "@stoplight/elements-starter-react",
+  "version": "1.0.0",
   "private": true,
+  "description": "Basic React app with embedded Stoplight Elements: StoplightAPI and Stoplight Project",
+  "author": "Stoplight <support@stoplight.io>",
+  "license": "Unlicensed",
```

Comment:
❗ The SPDX identifier of *The Unlicense* is `Unlicense`. 

Check out the table here: https://spdx.org/licenses/

Unlicensed is the exact opposite, that'd mean we don't release it under any license (aka *all rights reserved*).